subscriptions/views.py

```diff
@@ -193,8 +193,9 @@ def _update_subscription(self, form_data):
                     new_price = 0
                 elif form_data['coupon'] == 'charitable50':
                     new_price /= 2
+        for p in (*settings.PRICING, *settings.OLD_PRICING):
```

Comment:
This seemed to match the existing code most, though I guess, we could just take the amount from the self.object.price regardless of it existing in the settings...?